### PR TITLE
Final branding for 17.11

### PR DESCRIPTION
**Pull request by AR-May**
Final branding for 17.11

Also updated the public API version.

### CODE REVIEW

**Comment by rainersigwald** [comment]
Oh no, the `Check Version Bump On Release Branches` doesn't detect this case 😿

File: eng/Versions.props [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.11.0</VersionPrefix>
-    <PackageValidationBaselineVersion>17.8.3</PackageValidationBaselineVersion>
+    <VersionPrefix>17.11.0</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```
Comment (f-alizada) [inline eng/Versions.props:5]: Could you please add this to the new line? 
Also in order to understand: What exactly it does?

**Review by f-alizada — APPROVED** [review]

File: eng/Versions.props (hunk 1 above)
Comment (AR-May) [reply eng/Versions.props:5]: This is specifically a place where we DO NOT put a new line - for getting a merge conflict when we get it flowing to main branch. 
After adding this flag, we always need to bump a version if we need to merge anything, and the version considered stable.

**Comment by AR-May** [comment]
It seems like the build check test "CustomAnalyzerTest" started to fail 100% of time. I will disable it here and in the main.

**Review by rainersigwald — APPROVED** [review]

**Pull request merged**